db/migrate/20160502145559_remove_technology_id_field_from_importable_attributes.rb

```diff
@@ -0,0 +1,7 @@
+class RemoveTechnologyIdFieldFromImportableAttributes < ActiveRecord::Migration
+  def change
+    remove_index :importable_attributes, :technology_id_and_name
+    remove_column :importable_attributes, :technology_id
+    add_index :importable_attributes, [:technology_key, :name], unique: true
```

Comment:
Yes and no. Yes, because it's a good idea. But no because I'm currently trying to move all technologies to yaml files and in that case I'm getting rid of the `importable_attributes` table and therefor this file.